draft-ietf-masque-h3-datagram.md

```diff
@@ -249,23 +141,29 @@ corresponding stream.
 # Capsules {#capsule}
 
 This specification introduces the Capsule Protocol. The Capsule Protocol is a
-sequence of type-length-value tuples that allows endpoints to reliably
-communicate request-related information end-to-end, even in the presence of
-HTTP intermediaries.
-
-
-## Capsule Protocol {#capsule-protocol}
+sequence of type-length-value tuples that new HTTP Methods or new HTTP Upgrade
+Tokens can choose to use. It allows endpoints to reliably communicate
+request-related information end-to-end on HTTP request streams, even in the
+presence of HTTP intermediaries. The Capsule Protocol can be used to exchange
+HTTP Datagrams over versions of HTTP that do not support the QUIC DATAGRAM frame.
 
 This specification defines the "data stream" of an HTTP request as the
 bidirectional stream of bytes that follow the headers in both directions. In
 HTTP/1.x, the data stream consists of all bytes on the connection that follow
 the blank line that concludes either the request header section, or the 2xx
-(Successful) response header section. In HTTP/2 and HTTP/3, the data stream of
-a given HTTP request consists of all bytes sent in DATA frames with the
+(Successful) response header section. In HTTP/2 and HTTP/3, the data stream of a
+given HTTP request consists of all bytes sent in DATA frames with the
 corresponding stream ID. The concept of a data stream is particularly relevant
```

Comment:
Editorial: you might be able to strengthen the definition of data stream a bit by extracting the semantic from version concerns. For example, something like

"This specification defines the "data stream" of an HTTP request as the
bidirectional stream of bytes that follow the header section in both directions. 
The concept of a data stream is particularly relevant for methods such as 
CONNECT. In the client-to-server direction, the data stream always exists.
In the server-to-client direction, the data stream only exists if when there
is a 2xx (Successful) response. In HTTP/1.x, the data stream consists of all
bytes on the connection that follow the blank line concluding header section.
In HTTP/2 and HTTP/3, the data stream of a given HTTP request consists
of all bytes sent in DATA frames with the corresponding stream ID. "

Reply:
Let's tackle that in a follow-up since the design team isn't touching the concept of the data stream